airflow/contrib/operators/kubernetes_operator.py

```diff
@@ -495,12 +492,12 @@ def create_job_yaml(self, context):
                 'airflow-{}'.format(
                     configuration.get('core', 'environment_suffix')),
                 'labels': {
-                    'dag_id': self.dag_id,
-                    'execution_date': context['execution_date'].strftime('%Y-%m-%dT%H.%M.%S.%f'),
-                    'app.kubernetes.io/name': self.dag_id or UNDEFINED_LABEL,
-                    'app.kubernetes.io/part-of': self.dag.part_of or UNDEFINED_LABEL,
-                    'k8s.bluecore.com/team': self.dag.team or UNDEFINED_LABEL,
-                    'app.kubernetes.io/component': self.dag.component or UNDEFINED_LABEL,
+                    'dag_id':
+                    self.dag_id,
+                    'execution_date':
+                    context['execution_date'].strftime('%Y-%m-%dT%H.%M.%S.%f'),
+                    'partner':
+                    self.partner,
```

Comment:
@dangermike is this the right location for `partner` and why is it right / wrong?

Reply:
We want the partner to be a label on the pods and should have the `k8s.bluecore.io` prefix.